packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -83,23 +83,23 @@ export default {
          * @param name
          */
         selectTabIndex (name) {
+            if (this.activeTab === name) return;
+
             const previousIndex = this.tabs.findIndex(t => t.name === this.activeTab);
             const newIndex = this.tabs.findIndex(t => t.name === name);
 
             if (newIndex > previousIndex) {
-                this.direction = DIRECTION.RIGHT;
-            } else {
                 this.direction = DIRECTION.LEFT;
+            } else {
+                this.direction = DIRECTION.RIGHT;
```

Comment:
minor suggestion:
```
this.direction = newIndex > previousIndex ? DIRECTION.LEFT : DIRECTION.RIGHT;
```

Reply:
I did try this (who doesn't like more concise code? 😄) but thought I'd keep it the same because it's _slightly_ easier to read

Reply:
I used the need to run another build as an excuse to update this 👍🏻